changes/2818.bugfix.rst

```diff
@@ -0,0 +1 @@
+When no content is provided in the Window object, default to empty Box.
```

Comment:
A Window is now guaranteed to have a content widget. If no content is provided at time of construction, an empty Box widget will be added as the content.